RELEASE.md

```diff
@@ -1,3 +1,34 @@
+# Release 2.2.1
+
+This patch release includes a rollup to HEAD so as to support required features
+for uploading graphs to TensorBoard.dev.
+
+## TensorBoard.dev updates
+
+- TensorBoard.dev now renders model graphs, for experiments uploaded starting
+  from this release.
+
+## Features
+
+- TensorBoard.dev graph rendering support
+  - The `tensorboard dev upload` subcommand now sends the model graph, when
+    available, so that it can be rendered via the Graphs plugin on
+    TensorBoard.dev.  Large node attribute values (which would not be rendered
+    anyway) are filtered out before upload.  Graphs that remain larger than 10MB
+    after filtering are not uploaded.
+  - The `tensorboard dev list` subcommand now reports the total size of stored
+    binary objects (e.g., graphs) for each experiment (#3464) The `tensorboard
+    dev list` subcommand now accepts a `--json` flag to allow parsing the output
+    more easily (#3480)
+- Auto-reload is now disabled when the browser tab is not visible, saving
+  resources (#3483)
+- HParams plugin: substantial efficiency improvements (#3438, #3449)
```

Comment:
I'd omit this since these improvements won't really make much of a difference for most OSS TensorBoard users, so I wouldn't want to oversell this as a feature.